### ISSUE TITLE
[DEVICE REQUEST] Hue Dimmer switch 2021 version

### ISSUE THREAD

**Issue by DanteTheDonkey** [dresden-elektronik/phoscon-app-beta#367]
Hi,

Can you please add the new RW0022 Hue Dimmer Switch to Phoscon app? Deconz can see the device, but unable to add via Phoscon in order to use it in Homebridge at the moment.

This one issue aside, loving the ConbeeII and the software. So many devices now playing together nicely! Thank you

Many thanks!

**Comment by alefab** [comment]
Should be RWL022

**Comment by YKO-de** [comment]
It should already work under phoscon.de/pwabeta.

**Comment by eakoning** [comment]
Should I be able to see the remote represented here as well?
If I select the left one (which is the v1 version of the switch), pairing does not work.
It does show up in deCONZ but not in Phoscon. I'm not using the _pwabeta_ .
![image](https://user-images.githubusercontent.com/1950705/116871562-bd4a0400-ac14-11eb-92f5-449e88acf956.png)

**Comment by alefab** [comment]
> It should already work under phoscon.de/pwabeta.

The UI is displaying the RWL021 and not the RWL022 ( "Hue" text instead of the 0 ) even in the beta version.
With the current phoscon version, the RWL022 is detected and available in the old GUI ( Open Wireless Light Control (2016) ) but not in the pwa version.

**Comment by YKO-de** [comment]
The switch should be visible in the phoscon beta by now. However, it is currently interleaved with the image of RWL021, as there is no image for it yet.

**Comment by alefab** [comment]
Yes, I can confirm that I was able to see it in phoscon beta and asign functions to buttons

**Comment by eakoning** [comment]
Any idea when this will be available in the stable version?

**Comment by KennethLavrsen** [comment]
This is still not fixed

hey guys. The Phoscon you can run online from Dresden Elektronik has supported this new dimmer for several months. Why isn’t this merged into the production Phoscon that you ship? Is seems like a football just at your right foot and you are right in front of the goal.

**Comment by SEK-DE** [comment]
Our last stable was from the 17th or may. So please be patient, it will be integrated in the next.
@KennethLavrsen we don´t have a device called phoscon. Thats our firm name. 
The switch works in the latest beta, not in the stable. All of our new devices going out with 2.12.0 atm. 
So, none of them should be able to work with the switch, exept you go to the web-app beta.
Everything is like discribed.

**Comment by dustbox84** [comment]
Hi, I have tested the new beta and now i can find the philips switch. But in the switch editor  the switch is not visible to associate with a light.
<img width="1258" alt="Schalter auswählen" src="https://user-images.githubusercontent.com/79605605/126221298-691e59d7-d0a5-443a-acdc-2742b25a4187.png">
<img width="625" alt="Hersteller" src="https://user-images.githubusercontent.com/79605605/126221314-0a415882-41e8-4019-93de-9e61f4dd8cc0.png">

**Comment by pimw1** [comment]
Hi all, to be honest, i am a little disappointed with the througput time. Hue/Signifiy is the number one in the industry. Them releasing a new dimmer switch is quite a big thing. It is more than 6 months and it is still not implemented. If support goes so slow with the biggest brand, i can't even imagine how slow it will be for minor brands.

I really hope that you will improve upon this.

**Comment by SEK-DE** [comment]
@pimw1 thank you for the feedback. Just fyi: we are a really small team with hundrets of devices and features that customers want to have integrated every month plus business inquiries. In this special case, the device causes some unexpected problems who take a whil to solve. We are definately on it and hit you up if there is something new.

**Comment by pimw1** [comment]
Thank you for your answer. From the discord announcements, I read an old post about moving device configuration from C++ to JSON Device Description Files. Given the situation described (way more device requests than colleagues to process them), perhaps the community can help here? I'm not here to bash, I understand your situation and really appreciate the awesome stuff that you are doing. Whatever can be done to improve things, would be more than welcome. I'm curious about your thoughts.

**Comment by SEK-DE** [comment]
Don´t get me wrong, i´m not angry or anything. We here that very often. 
So far the community and the CC´s are doing a fantastic job and we will go to release stable versions more often in the future, so that every user of HA integrations will get there wished device sooner. There will be allways device who causes us headache, but thats the way i guess 😉

**Comment by KennethLavrsen** [comment]
Go away bot

**Comment by pimw1** [comment]
The latest version of Phoscon supports the Hue Dimmer switch 2021 version.

**Comment by KennethLavrsen** [comment]
Yes. The only snall cosmetic detail is that the image in the UI still shows the old. I do not personally care. But it is still an open detail

**Comment by pimw1** [comment]
Definitely.

**Comment by alefab** [comment]
In fact, in the  2.12.06 version, the switch is available in the "switches"
devices page, but I can not choose it in the light edit when I want to add
the switch to the group. And I can not use the beta version anymore, I have
a login issue trying to use the pwa beta version.

**Comment by eakoning** [comment]
I'm experiencing the same problem as @dustbox84 and @alefab. Switch editor cannot be used. Is this something that needs a vNext version or am I missing an important step? The dimmer is recognized in the Switch section but still showing image of the old dimmer.

![image](https://user-images.githubusercontent.com/1950705/135897138-007efcd6-d310-49ff-8e5b-85d5eb533dbc.png)

@YKO-de mentioned in May that assigning buttons was possible in the beta.
My HA installation is running the deCONZ integration 6.10.0 (using deCONZ 2.12.6 under the hood).

**Comment by DukyNuky** [comment]
> I'm experiencing the same problem as @dustbox84 and @alefab. Switch editor cannot be used. Is this something that needs a vNext version or am I missing an important step? The dimmer is recognized in the Switch section but still showing image of the old dimmer.
> 
> ![image](https://user-images.githubusercontent.com/1950705/135897138-007efcd6-d310-49ff-8e5b-85d5eb533dbc.png)
> 
> @YKO-de mentioned in May that assigning buttons was possible in the beta. My HA installation is running the deCONZ integration 6.10.0 (using deCONZ 2.12.6 under the hood).

can you tell me how ? i have the same problem. 2.12.06 i can add the switch. But can not assign it to a group

**Comment by adibis** [comment]
Same issue here. Switch was added to phoscon but there are no entities other than battery level. I cannot assign switch to any group (doesn't even show up).  Running 2.12.06 version.

**Comment by hoblins** [comment]
Unfortunately I can still confirm this issue. Using docker-deconz stable as well as latest.
Any feedback or even bugfixing from the developers would be appreciated.

By the way: Can the phoscon beta app somehow be used with the docker image?

**Comment by DukyNuky** [comment]
hi, i solved the problem. 

use http://phoscon.de/pwabeta to setup the switch. if you cant login.. use different browser. Firefox is working here. Chrome is blocking something, so you cant login.

**Comment by hoblins** [comment]
Thank you for that.

Yes, the switch can be selected now in a group. But it is displayed as another switch type:
![grafik](https://user-images.githubusercontent.com/11378643/140618055-e948abe3-a102-4de4-bdb9-b8d844e8ea71.png)

By the way, with Firefox it is not working for me. After logging in it redirects to the login page again.
But with Chrome it works with this workaround:
[https://forum.phoscon.de/t/password-stopped-working-cannot-reset/357/13](url)

**Comment by Sokratesli** [comment]
same issue here. The RWL022.04 is listed under the switches. But if I would like to add the switch to a group it is not selectable. thank you for your help!

**Comment by camilolunacom** [comment]
Any updates on this one? Has anyone been able to solve the issue?

**Comment by ulbrich-media** [comment]
I had the same issues with the Phillips Hue Dimmer Switches, but everything works fine now. All I needed to do was to update to the latest deCONZ Docker image. There is a new one out there [deconzcommunity/deconz](https://github.com/deconz-community/deconz-docker) which replaces the old `marthoc/deconz`, so you may need to change that, too.

**Comment by flixha** [comment]
I'm having the same issue as @hoblins that Phoscon displays the wrong type of 4-button square switch when trying to assign the buttons of the RW0022 Hue Dimmer Switch via a group's switch editor in the beta version of the Phoscon app.

I am able to assign the 3 actual top buttons of the dimmer switch (correspond to top left, top right, bottom left in the app). But when I try to select the bottom right button (which hopefully should be the "Hue" button on the switch), then once the assignment is done the Phoscon app suddenly switches the function from the bottom right to the bottom left button:

![phoscon_01](https://user-images.githubusercontent.com/16126887/148677499-b0648cf2-fab0-411b-a85c-ee3ec6b06ffc.png)
![phoscon_02](https://user-images.githubusercontent.com/16126887/148677507-c2c946cf-126c-4383-9f74-39ae7a08ae5c.png)
![phoscon_03](https://user-images.githubusercontent.com/16126887/148677510-734267d7-053b-425b-a157-c0feb99677df.png)


> I had the same issues with the Phillips Hue Dimmer Switches, but everything works fine now. All I needed to do was to update to the latest deCONZ Docker image. 

Which version number can you see in the Phoscon app @ulbrich-media ?
I'm running the Deconz Raspbian Image and Phoscon shows version 2.13.04, so I thought I should be running the latest version...

**Comment by ulbrich-media** [comment]
Im running on 2.13.04 (18/12/2021) as well. However, I'm running on Docker which shouldn't make any difference I guess. 

For me everything looks like this:
![Screenshot 2022-01-09 at 12 44 42](https://user-images.githubusercontent.com/27144164/148680973-dedce410-a303-43f9-a8e3-8fad73f4e709.png)
![Screenshot 2022-01-09 at 12 45 09](https://user-images.githubusercontent.com/27144164/148680974-50fe0639-9537-435d-b0c8-390280c962a2.png)
